server/views/temporary-accommodation/static/useNDelius.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Use nDelius" %}
```

Comment:
Suggest this should be:
`{% set pageTitle = applicationName + " - Referrals should be submitted through nDelius" %}`
to match conventions elsewhere

Reply:
:recycle: